### PR TITLE
dev version of coq-relation-algebra

### PR DESCRIPTION
**Pull request by damien-pous**
cf https://github.com/damien-pous/relation-algebra/issues/17

### CODE REVIEW

File: extra-dev/packages/coq-relation-algebra/coq-relation-algebra.dev/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  "keyword:matrices"
+  "category:Mathematics/Algebra"
+  "logpath:RelationAlgebra" ]
+authors: [
+  "Damien Pous <Damien.Pous@ens-lyon.fr>"
+  "Christian Doczkal <christian.doczkal@ens-lyon.fr>"
+]
+url {
+  src:
+    "https://github.com/damien-pous/relation-algebra.git#master"
```
Comment (anton-trunov) [inline extra-dev/packages/coq-relation-algebra/coq-relation-algebra.dev/opam:36]:
```suggestion
    "git+https://github.com/damien-pous/relation-algebra.git#master"
```

File: extra-dev/packages/coq-relation-algebra/coq-relation-algebra.dev/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+synopsis: "Relation Algebra and KAT in Coq"
+maintainer: "Damien Pous <Damien.Pous@ens-lyon.fr>"
+version: "dev"
```
Comment (anton-trunov) [inline extra-dev/packages/coq-relation-algebra/coq-relation-algebra.dev/opam:4]:
```suggestion
version: "dev"
```

**Review by anton-trunov — COMMENTED** [review]
Thanks @damien-pous! I just left a couple suggestions to make the opam file look more like the rest of them in this repo.

**Comment by palmskog** [comment]
Looks like there is some incompatibility with Coq `master`:
```
# File "ra_fold.mlg", line 89, characters 8-24:
# Error: The constructor Case expects 7 argument(s),
#        but is applied here to 5 argument(s)
```
If this is expected I can still merge the PR.

**Comment by damien-pous** [comment]
thanks @anton-trunov for the suggestion, and help!

**Pull request merged**